README.rst

```diff
@@ -1,7 +1,12 @@
 About the AnacondaRecipes/aggregate repository
 ==============================================
 
-The aggregate repository is roughly similar to the concept of the ``conda-forge`` feedstocks repository. It is meant to operate as a centralization of many external repositories, with each of those external repositories following the one-recipe-per-repo concept. We fork feedstocks from ``conda-forge``, and the long-term goal is to keep in sync with ``conda-forge`` recipes. Unlike the ``conda-forge`` feedstocks repository, aggregate also contains some recipes directly in the aggregate repo. We consider this a temporary staging area for recipes that eventually need to be submitted to ``conda-forge``, or for recipes that we think ``conda-forge`` will have no interest in.
+The aggregate repository contains:
+
+- The definition of globally pinnend packages: `conda_build_config.yaml <https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml>`_ (similar to the `conda-forge-pinnings-feedstock <https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml>`_)
+- Submodule definitions to centrally reference all public maintained individual feedstocks (one recipe per repository) with relativ repo URL and their release branches: `.gitmodules <https://github.com/AnacondaRecipes/aggregate/blob/master/.gitmodules>`_. (similar to ``conda-forge`` feedstocks repository)
+- The submodule pinning to a sha1 of the invidual external repository.
+- Staging area for recipes that eventually need to be submitted to ``conda-forge``, or for recipes that we think ``conda-forge`` will have no interest in.
```

Comment:
My criticism of removing the "story" part is that this is now written for an audience of people who already know quite a lot about conda and git. For example what is a "pinning"? Is it something mentioned in cbc.yaml or is it a git submodule hash? I understand perfectly what you mean but I think people with less experience won't.

Some suggestions off the top of my head:

- first point is mostly fine, I would mention when cbc.yaml is used though or link to docs.
- for the rest, I don't think it's clear that the aggregate is just a list of feedstocks. I would do something like
  - aggregate is list of feedstocks
    - most feedstocks are submodules, with the submodule hash pointing to latest published commit
      - mention important issues - relative url, branch
      - mention problems keeping this up to date, mention git submodule update option to fetches latest of module branch (can't remember exactly what it is at the moment)
    - some feedstocks are normal directories checked into aggregate
      - staging area, etc

Reply:
Updated and hopefully covered all items.